spec/lib/audit/catalog_to_moab_class_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
-require_relative '../../../lib/audit/catalog_to_moab.rb'
+require 'audit'
```

Comment:
can we just `require 'audit' ` in `rails_helper.rb`?

Reply:
In fact, I think we might not even have to require anything since we can rely on Rails auto/eager-loading.